Tools-Override/XUnitNetFXConfigFileTemplate.txt

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<configuration>
+   <runtime>  
+      <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1"> 
+
+         <dependentAssembly>  
+            <assemblyIdentity name="xunit.execution.dotnet" publicKeyToken="8d05b1bb7a6fdb6c" culture="neutral" />
+            <codeBase version="2.2.0.3300" href="{RunTimePath}xunit.execution.dotnet.dll"/>
+         </dependentAssembly>
+
+         <dependentAssembly>  
+            <assemblyIdentity name="System.IO.FileSystem" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
```

Comment:
talked offline with @weshaggard and we are going to automatic generate the list in a future PR